packages/block-library/src/editor.scss

```diff
@@ -55,22 +55,8 @@
 @import "./post-featured-image/editor.scss";
 @import "./post-comments-form/editor.scss";
 @import "./post-content/editor.scss";
-
 @import "./editor-elements.scss";
 
-:root .editor-styles-wrapper {
-	@include background-colors-deprecated();
-	@include foreground-colors-deprecated();
-	@include gradient-colors-deprecated();
-}
```

Comment:
This style is not relevant to the problem we are trying to solve, so we may want to keep it just in case. It might be possible to remove it in the future. What do you think?

Reply:
Good call. I'll add it back.